src/app/directives/common/new-feature-indicator/new-feature-indicator.directive.ts

```diff
@@ -6,6 +6,16 @@ import { NewFeatureIndicatorWrapperComponent } from 'app/directives/common/new-f
 import { NewFeatureIndicator } from 'app/directives/common/new-feature-indicator/new-feature-indicator.interface';
 import { NewFeatureIndicatorService } from 'app/directives/common/new-feature-indicator/new-feature-indicator.service';
 
+/**
+ * Usage: adding an indicator with a hint about a new feature.
+ *
+ * @example
```

Comment:
This is good to have but I meant an example to use new directive somewhere in the project, so it can be tested via docker image. Otherwise it's okay if no plan to use it now.